src/client/pages/Debugger/components/Offer.tsx

```diff
@@ -90,9 +108,26 @@ const getCurrentAvailableQuoteData = (
   return currentQuoteTypeData
 }
 
+const getDanishQuoteValues = (
+  values: Values,
+  quoteType: 'danishTravel' | 'danishAccident',
+) => {
+  const { danishHomeContents, ...filteredValues } = values!
+  const { type, livingSpace, ...QuoteTypeValues } = danishHomeContents!
```

Comment:
OK this might be a bigger discussion on how we use TypeScript but to me it feels like something's actually off with the typing, which should probably be properly adressed by creating new types or whatever, if one needs to use non-null assertion and type assertion as much as we do in this file now 🙈 But I can of course be wrong and this is the only way to do it.